Automattic/woocommerce-payments
# Add E2E screenshots saved to file

#### Changes proposed in this Pull Request

Take screenshots during E2E test run, and store as artifact ([retained for 90 days](https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration#artifact-and-log-retention-policy)).

#### Testing instructions

Verify that a set of screenshots appears in the `/screenshots` directory after the E2E test run, and are accessible as a downloadable artifact from the workflow run (see one of the E2E checks below » Details » Artifacts at top).

-------------------
